core/drivers/fabric-driver/server/dbConnector.ts

```diff
@@ -53,15 +55,20 @@ class LevelDBConnector implements DBConnector {
         }
         this.DB_NAME = dbName;
         this.dbHandle = new Level(path.join(process.env.DB_PATH ? process.env.DB_PATH : "./driverdbs", dbName), { valueEncoding: 'json' });
+        // Retry max attempts, default 250, making it 5 seconds for retries
+        this.dbOpenMaxRetries = process.env.LEVELDB_LOCKED_MAX_RETRIES ? parseInt(process.env.LEVELDB_LOCKED_MAX_RETRIES) : 250;
+        // Retry back off time in ms, default 20ms
+        this.dbRetryBackoffTime = process.env.LEVELDB_LOCKED_RETRY_BACKOFF_TIME ? parseInt(process.env.LEVELDB_LOCKED_RETRY_BACKOFF_TIME) : 20;
     }
 
     async open(
         i: number = 0
     ): Promise<boolean> {
+        logger.debug(`attempt #${i} to open database ${this.DB_NAME}`)
         try {
             await this.dbHandle.open();
         } catch (error: any) {
-            if (i>=10) {
+            if (i>=this.dbOpenMaxRetries) {
```

Comment:
Put white spaces in the expressions, which is generally our convention: `i >= this.dbOpenMaxRetries`.